### PR TITLE
Change demo to use routing.js

### PR DESCRIPTION
**Pull request by EWhite613**
# minor

Allows for simpler porting with frost guide. Most likely going to change the routing.js but I need to test it first

### CODE REVIEW

File: tests/dummy/app/router.js [hunk 1]
```diff
@@ -5,13 +5,50 @@ var Router = Ember.Router.extend({
   location: config.locationType
 })
 
+var addRoute = function (routeConfig) {
```
Comment (sglanzer-fire) [inline tests/dummy/app/router.js]: Can we put this function into an addon so that we can use it in multiple places?

Comment (EWhite613) [inline tests/dummy/app/router.js]: Ya I can do that

**Comment by EWhite613** [comment]
@sglanzer add addon for function

**Comment by sglanzer-fire** [comment]
Approved - nice, much more reusable, thanks!

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-link/pull-request/8/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-link)

**Pull request merged**